notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1593_force_actuator_error_elevation.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.        fadata_min["FA"].append(f"Force Actuator {i}")
</pre><p>I recommend grouping the code here. Something like:</p>

<pre class="ql-syntax" spellcheck="false">for ...
    (optional empty line)
&nbsp; &nbsp; fadata_mean["elevation"].append(bin_centers)
&nbsp; &nbsp; fadata_mean["force error"].append(fa_mean[i])
&nbsp; &nbsp; fadata_mean["FA"].append(f"Force Actuator {i}")
&nbsp; &nbsp; 
    fadata_max["elevation"].append(bin_centers)
&nbsp; &nbsp; fadata_max["force error"].append(fa_max[i])
&nbsp; &nbsp; fadata_max["FA"].append(f"Force Actuator {i}")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Sorry, I don't quite understand this comment. How is this any different to what is there already? There is a for loop filling in all these dictionaries as you suggest.</p>